<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Petr Vanek &lt;petr@scribus.info&gt;</author>
 <comment>This is base class for Scribus Barcode Generator. Always use designer for this file!</comment>
 <class>BarcodeGeneratorBase</class>
 <widget class="QDialog" name="BarcodeGeneratorBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Barcode</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QGroupBox" name="bcodeBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="6" column="1">
       <widget class="QCheckBox" name="includecheckintextCheck">
        <property name="toolTip">
         <string>Include the checksum digit in the barcode text</string>
        </property>
        <property name="text">
         <string>Display checksum</string>
        </property>
        <property name="shortcut">
         <string>Alt+U</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="includetextCheck">
        <property name="toolTip">
         <string>If checked, there will be numbers in the barcode too</string>
        </property>
        <property name="text">
         <string>Show readable text</string>
        </property>
        <property name="shortcut">
         <string>Alt+I</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QCheckBox" name="parsefncCheck">
        <property name="text">
         <string>Parse specials</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="guardwhitespaceCheck">
        <property name="toolTip">
         <string>Draw arrows to be sure of space next the code</string>
        </property>
        <property name="text">
         <string>Guard whitespace</string>
        </property>
        <property name="shortcut">
         <string>Alt+G</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QCheckBox" name="parseCheck">
        <property name="text">
         <string>^NNN as ASCII</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="includecheckCheck">
        <property name="toolTip">
         <string>Generate and include a checksum in barcode</string>
        </property>
        <property name="text">
         <string>Add checksum</string>
        </property>
        <property name="shortcut">
         <string>Alt+N</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QGroupBox" name="colorBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="topMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="bgColorButton">
        <property name="toolTip">
         <string>Background color - under the code lines</string>
        </property>
        <property name="text">
         <string>&amp;Background</string>
        </property>
        <property name="shortcut">
         <string>Alt+B</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="bgLabel">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="lnColorButton">
        <property name="toolTip">
         <string>Color of the lines in barcode</string>
        </property>
        <property name="text">
         <string>&amp;Lines</string>
        </property>
        <property name="shortcut">
         <string>Alt+L</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="linesLabel">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="txtColorButton">
        <property name="toolTip">
         <string>Color of the text and numbers</string>
        </property>
        <property name="text">
         <string>&amp;Text</string>
        </property>
        <property name="shortcut">
         <string>Alt+T</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="txtLabel">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>142</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="bcFamilyLabel">
        <property name="text">
         <string>Barcode Family:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="bcFamilyCombo"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="bcLabel">
        <property name="text">
         <string>&amp;Barcode:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>bcCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="bcCombo">
        <property name="toolTip">
         <string>Select one of the available barcode types here</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="codeLabel">
        <property name="text">
         <string>&amp;Contents:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>codeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="codeEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="toolTip">
         <string>The contents encoded in the barcode</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="resetButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Reset the barcode samples</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="optionsLabel">
        <property name="text">
         <string>Options:</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="optionsEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Advanced options for the barcode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QGroupBox" name="optionsBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="2" column="0" colspan="5">
       <widget class="QLabel" name="sampleLabel">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>190</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : #FFF</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="formatLabel">
        <property name="text">
         <string>Version:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>formatCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="formatCombo">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>1</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Version of the symbol</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QComboBox" name="eccCombo">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Error correction level</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="eccLabel">
        <property name="text">
         <string>Error correction level:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>eccCombo</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>bgColorButton</tabstop>
  <tabstop>lnColorButton</tabstop>
  <tabstop>txtColorButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
